tests/cmd/stability/main.go

```diff
@@ -407,7 +409,7 @@ func newTidbClusterConfig(ns, clusterName string) *tests.TidbClusterConfig {
 		TiKVImage:        fmt.Sprintf("pingcap/tikv:%s", tidbVersion),
 		TiDBImage:        fmt.Sprintf("pingcap/tidb:%s", tidbVersion),
 		PumpImage:        fmt.Sprintf("pingcap/tidb-binlog:%s", tidbVersion),
-		StorageClassName: "local-storage",
+		StorageClassName: storageclassName,
```

Comment:
Just remove the settings, no need to set the sc explicitly, and it will use the `default` sc automatically.